src/libraries/System.IO.FileSystem/src/System/IO/FileStatus.Unix.cs

```diff
@@ -227,6 +227,8 @@ private DateTimeOffset UnixTimeToDateTimeOffset(long seconds, long nanoseconds)
 
         private unsafe void SetAccessOrWriteTime(string path, DateTimeOffset time, bool isAccessTime)
         {
+            //only used for access time on OSX (unless being used as a fallback), used for both on other unix platforms
```

Comment:
```suggestion
            // Only used for access time on OSX (unless being used as a fallback), used for both on other Unix platforms
```

Reply:
I'll put this in the next commit so it doesn't build twice.